packages/Meshfree/src/DTK_DetailsSVDImpl.hpp

```diff
@@ -276,9 +271,7 @@ struct SVDFunctor
             givens_left( V, cr, sr, p, q );
 
             norm = norm_F_wo_diag( E );
-            num_iter++;
         }
-        assert( num_iter < max_iter );
```

Comment:
@Rombur or @aprokop  Please comment on the removal of `max_iter` here

Reply:
I don't know why it was there in the first place. It was copied from Minitensor that had it, but SVD is deterministic, and just has to go through off-diagonal nonzeros. I had a comment that it should be possible to remove, and @Rombur did that.

Reply:
When I added the test for SVD, the test was failing because we hit `max_iter` before convergence. Since there was no reason to have it, I removed it.

Reply:
Fair enough